server/src/cm_job_task.cpp

```diff
@@ -3089,7 +3089,7 @@ tsRenameDB (nvplist * req, nvplist * res, char *_dbmt_error)
     argv[argc++] = newdbname;
     argv[argc++] = NULL;
 
-    snprintf (task_name, TASKNAME_LEN, "%s", "renamedb");
+    strcpy(task_name, "renamedb");
```

Comment:
It would be great to follow the coding style:
```c
  strcpy (task_name, "renamedb");
```

I also hope we will use a code formatter.